website/profile/utils.py

```diff
@@ -31,15 +31,14 @@ def get_gravatar(user, size=None):
     )
 
 
-def serialize_user(user, node=None, full=False):
+def serialize_user(user, node=None, admin=False, full=False):
```

Comment:
Is it ever possible for this function to be called with a `user` who is a contributor to `node` and `admin=True`?